lizard_downloader.py

```diff
@@ -209,12 +204,15 @@ def show_data(self):
 
     def show_data(self):
         """Show the data as a new layer on the map."""
+        # Set the status bar text
+        status_bar_text(self, "Downloading {}...".format(ASSET_TYPES[0]))
         # Get a list with JSONs containing the data from the Lizard API
         payload = {"page_size": 100}
         list_of_assets = get_data(ASSET_TYPES[0], payload)
 
         # Create a new vector layer
         self.layer = create_layer(ASSET_TYPES[0], list_of_assets)
 
-        # # Close the lizard_downloader_dialog
-        # self.dlg.close()
+        # Set the status bar text
+        asset_type_caps = ASSET_TYPES[0][0].capitalize() + ASSET_TYPES[0][1:]
```

Comment:
What's happening here? ``capitalize()`` already capitalizes the first letter, so shouldn't that be enough?